### PR TITLE
Python docs

### PR DESCRIPTION
**Pull request by swinney**
Update Python docs to fit standard User_Docs / User_Code separation.

### CODE REVIEW

File: Languages/Python/Example2/Example3/README.md
Comment (paulasanematsu) [inline Languages/Python/Example2/Example3/README.md:1]: Did you move Example3 folder inside Example2 folder on purpose? Or did you intent to simply rename Example3 -> Example2?

Comment (swinney) [reply Languages/Python/Example2/Example3/README.md:1]: Yes, I removed Example2 and moved Example3 ( renamed ) to Example2.  I also updated the README accordingly.

Comment (swinney) [reply Languages/Python/Example2/Example3/README.md:1]: Oh crud, I see what happened.  Don't know how I didn't catch that.

Comment (swinney) [reply Languages/Python/Example2/Example3/README.md:1]: Fixed.

**Comment by paulasanematsu** [comment]
All good from my point of view.

**Pull request merged**